### ISSUE TITLE
[FEATURE] create issues for TODO items

### ISSUE THREAD

**Issue by clach04** [parazeeknova/nyxtext#13]
The readme has a number of outstanding TODO and Bug items.

Have you considered creating tickets for those and getting them linked?
E.g. https://github.com/parazeeknova/nyxtext/blob/3cfb7d72730acc4a16745d6a9967a3f9ba9ef19b/README.md?plain=1#L212

**Comment by parazeeknova** [comment]
Nope, i didn't know i could do that, I'll surely create tickets for those bugs and features soon! Thanks for the suggestion..
